spec

```diff
@@ -827,7 +827,6 @@ endseg
 beginseg
     name "ovl_Bg_Haka_Trap"
     include "build/src/overlays/actors/ovl_Bg_Haka_Trap/z_bg_haka_trap.o"
-    include "build/data/overlays/actors/z_bg_haka_trap.data.o"
     include "build/data/overlays/actors/z_bg_haka_trap.reloc.o"
```

Comment:
If the overlay is matching you can delete the old `.reloc.s` file in `data/overlays/` and use the reloc file generated by ZAP2.
```suggestion
    include "build/src/overlays/actors/ovl_Bg_Haka_Trap/ovl_Bg_Haka_Trap_reloc.o"
```